### DISCUSSION TITLE
添加Front-matter自定义多个子参数

### DISCUSSION THREAD

**Discussion by 0000rookie** [Qexo/Qexo#361] [Ideas]
在Qexo新建文章的时候，在右侧栏可以看到有标签添加多个子参数，比如下面这样
tags:
  - Linux
  - Android

但我们有时候也需要一些其他配置的自参数，想要添加上面像标签一样的就没办法添加了，我尝试了一下使用json，结果并不太友好，如果大佬可以添加一些这个功能就好了。
比如我有需求使用`pictures`这个参数，需要和tags一样有多个子参数。希望大佬可以看到，谢谢。

**Comment by am-abudu** [comment]
近期事情较多，会放到之后一起处理
